src/app/extensions/order-templates/shared/order-template-widget/order-template-widget.component.html

```diff
@@ -0,0 +1,36 @@
+<ish-info-box
+  heading="account.ordertemplates.widget.heading"
+  class="infobox-wrapper h-100"
+  cssClass="infobox-color-widget d-flex flex-column"
+>
+  <div class="loading-container">
+    <div *ngIf="!(loading$ | async); else loading" class="pb-2">
+      <ng-container *ngIf="orderTemplates?.length; else emptyList">
+        <div *ngFor="let orderTemplate of orderTemplates" class="mb-2">
+          {{ orderTemplate.title }}
+          <a *ngIf="orderTemplate.items?.length" class="align-top float-right">
+            <ish-product-add-to-basket
+              displayType="icon"
+              [product]="dummyProduct"
+              class="p-0 mb-0"
+              (productToBasket)="addTemplateToBasket(orderTemplate.id)"
```

Comment:
See order-template-widget.component.ts Why not call the method with the entire orderTemplate ?